cli/tests/services.bats

```diff
@@ -51,3 +51,19 @@ teardown() {
   RUST_LOG=flox=debug FLOX_FEATURES_SERVICES=true run "$FLOX_BIN" init
   assert_output --partial "service management enabled"
 }
+
+@test "services in the manifest" {
```

Comment:
I made this an "integration" test because by default Rust tests are run on separate threads in the same process, so setting an environment variable would affect _all_ unit tests.

Reply:
suggestion: impl as unit tests using `test_env`

Reply:
Moved to a unit test, the feature flag is no longer relevant for parsing the services section